UPGRADE.md

```diff
@@ -51,17 +51,18 @@ There is a list of all the repositories maintained by monorepo, changes in log b
 
 ## [From 7.0.0-beta1 to Unreleased]
 ### [shopsys/project-base]
-- [#497 adding php.ini to image is now done only in dockerfiles](https://github.com/shopsys/shopsys/pull/497)
+- *(optional)* [#497 adding php.ini to image is now done only in dockerfiles](https://github.com/shopsys/shopsys/pull/497)
     - you should make the same changes in your repository for the php.ini configuration files to be added to your Docker images
+        - remove all volumes from `docker-compose.yml` templates that include `php-ini-overrides.ini`
```

Comment:
does this mean all `docker-compose.yml.dist` ?

Reply:
yes, should I reformulate the sentence? Do You have suggestion maybe?

Reply:
yes it would be better if the all of the docker-compose.yml templates end with .dist .